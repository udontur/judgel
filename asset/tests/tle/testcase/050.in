3404 56
98900 35836 53135 11904 34850 86679 62892 87689 91907 10250 61119 85406 64195 26413 65608 1643 3433 32566 59571 73768 60744 34807 60885 41035 45942 31215 52954 94681 69605 89441 78823 35460 57115 95862 9711 95546 54429 8171 23959 31214 9546 43026 92423 40479 17884 9890 10987 97704 67324 38928 51966 20349 88965 39665 11736 85792 48420 14077 46938 81919 90232 87034 37677 19385 69037 63216 99823 45912 49657 83810 87751 91661 39574 44458 96331 88078 42682 66432 49731 53815 82128 60509 67615 69086 56979 26974 84815 70877 48839 19018 64366 21022 98221 31880 12178 56591 51005 3220 25969 73719 70452 91808 30852 35491 84604 78690 40546 34826 75419 56110 32759 99926 58078 61182 28654 32828 98946 52315 98728 32369 5678 44241 77972 97676 99891 40937 54165 69978 82204 95201 8881 51949 27132 92360 80876 62603 21689 87403 43471 77487 33756 39328 1301 39473 46563 40233 34308 41481 29947 94340 73734 31239 92157 21744 92059 51265 48942 33656 1045 30881 77020 2534 52405 66814 3249 93095 15599 49652 73836 23976 81442 86703 77737 58554 82 86430 67744 34138 20556 56967 81535 29834 16898 40731 70092 3117 77502 52454 91110 20657 92266 29426 98733 38842 447 61572 4169 14028 61216 18978 67340 2139 60979 93507 76382 51700 68003 82813 66020 32877 87690 25089 76484 28417 20027 11968 98218 834 94433 62004 89750 31239 95915 66402 16602 33516 6695 10546 91889 230 63122 8711 16310 42392 18110 16405 9581 35388 18060 41255 58666 76595 89360 4608 33851 70250 61417 80361 67474 70502 39248 8465 83045 36954 4254 90071 95948 75247 11917 22612 36722 2059 64952 67233 97453 74587 68502 6760 67053 4399 88696 40449 18829 55482 79024 25431 99267 35544 21345 96750 29751 45744 32188 74337 7684 97706 10506 30530 27944 86498 69829 2633 36627 9507 91617 14707 7659 53323 10726 84199 7690 1037 8937 16927 85054 98455 93722 47911 39911 88954 25710 529 36738 94191 36614 98827 89657 67944 11430 68432 60755 17285 55859 67928 45064 64393 69509 20742 40307 71235 76398 12323 7936 58695 8518 4717 85739 20890 71228 59780 72868 67448 25526 47418 22339 85087 26749 37020 52009 3884 45494 2745 95763 88818 44699 94608 5483 83057 17260 50494 47199 44868 37505 57041 28552 62066 51169 57865 24462 75073 88043 97035 65981 14350 3632 44321 95936 75696 99081 65174 57704 49779 98799 52136 4836 31941 20722 14818 38020 25622 57301 23219 13935 13748 82887 69420 35372 92570 27970 15656 9987 71183 45058 32759 39288 22249 45513 63900 98578 56025 67902 9454 90294 3632 39320 38209 46795 51807 13340 89603 12905 82598 1669 40530 15309 68524 45243 28905 37355 86131 62858 11991 30111 24308 67206 60057 31940 26292 70359 18282 90671 42085 50968 19153 4297 69589 44715 30822 81710 86464 22988 59982 82729 76305 92808 37795 38057 38431 13367 23198 35816 88360 61842 93197 59474 78619 6229 99277 1911 66980 66462 84599 95758 45579 1273 70109 14281 91276 18067 81997 54242 15603 27644 37836 74195 61455 97294 39640 4623 88991 74593 8130 90155 7139 42277 10055 97953 96070 8349 41244 32805 75599 32580 11903 59228 30823 97217 58720 15765 80452 68424 67138 26142 27426 77049 66622 76488 51306 11217 96542 92362 18647 33557 78182 572 59058 9593 78984 644 18826 62477 13610 60805 68730 60840 17878 37799 72085 49830 75573 80652 8946 82566 59181 39463 51892 26059 21729 22154 38508 72654 37117 27063 46423 53121 97498 71399 33459 21236 82260 25851 73815 1555 31582 14947 59441 36906 7371 98480 24068 99376 581 1612 37116 24847 47374 65176 86699 7731 12715 56013 72740 45720 32675 32100 27896 17062 97763 21885 28883 98089 54379 90766 50382 42281 41088 25550 52186 93515 37840 97026 87810 45794 94168 39947 82152 17936 78544 70880 34159 5300 4640 46447 64689 50889 66176 68745 90385 31710 29334 55077 54637 59143 35779 40526 64180 26198 35421 68472 96326 11587 4591 89273 55875 47259 3495 5779 74813 66964 74663 12969 22618 50569 60860 40546 83937 55798 11666 31060 89685 87551 86821 79627 59442 1434 76494 1874 93115 49879 64803 42668 24084 80504 93743 10709 13916 21093 69816 56531 14815 65026 58201 7348 84418 44638 21152 63096 76858 8910 52250 52510 85612 44576 88398 93397 4107 88352 88639 26799 38392 68750 46161 32169 28926 40027 12755 74737 68465 46268 66605 61717 85773 28140 95153 53663 97352 37539 78697 79198 13863 56714 70877 73966 17079 655 11219 34756 83705 58194 40163 72843 47883 75252 90369 80045 70498 11122 58410 67975 23409 70850 68887 17262 94744 7641 82229 15131 64382 21623 35120 95388 12808 77163 66776 22282 43917 29295 23347 76286 41468 2088 46347 812 92712 83016 18787 26170 11920 56507 35135 46327 98123 20976 36243 44918 42839 49419 40375 30576 47809 41253 78746 48427 40814 51826 2570 48619 72184 91389 96818 81763 43145 87720 44883 42062 9474 85884 239 52626 54695 43073 47351 16926 40140 2704 18681 59633 19936 49578 47332 51903 66373 77291 77819 63913 65494 71798 71129 50183 95734 15011 12113 52414 5207 16191 15173 80970 30539 29279 2315 51908 76762 54158 98130 91864 81595 53044 88683 59432 48258 73170 50358 52919 4865 34836 3945 65286 55729 67899 64754 15312 24908 31833 70586 94042 11143 8778 76733 40509 7347 49115 75861 31938 98038 84222 79520 49549 14600 31349 94122 67305 63799 50851 31422 39589 43179 77294 24927 4064 89671 54355 93595 79718 9005 53141 90370 34027 37893 71746 28865 52688 21587 68425 37267 71346 75247 77323 16633 50889 47807 86813 54017 49060 29424 547 60381 1783 5419 89926 43597 93394 21899 28231 21246 33081 1837 99942 49163 8128 47237 27136 52028 21608 76464 73137 58539 9469 41264 65209 28962 34986 77228 23633 3008 66167 70847 19387 50170 81806 60267 82245 72051 4485 43188 54287 56017 65537 9136 13944 43613 19270 72779 95606 62002 92592 4120 18883 96715 31366 86803 50447 47126 21855 30195 68039 71354 20493 69742 44773 33953 11846 83201 56026 99870 2086 81737 23329 77739 91980 1099 35517 91465 96979 34531 78609 56289 95330 8289 3825 38712 97837 56570 37908 22103 37305 76951 70995 44693 66199 75290 71021 6098 64727 35885 73097 37387 12665 90439 69698 74402 12187 46674 56918 34831 5120 8445 48307 9978 87490 74496 96887 2614 57065 97370 79548 1118 40866 20667 9117 89355 47951 53443 27800 13796 86772 31137 18973 63210 18057 50936 98395 91390 23488 71368 4580 10398 15324 3591 12492 96309 62167 97156 40513 25520 2660 87978 54777 14907 34275 44751 17996 81290 34347 13639 29932 89125 63720 73957 7132 29435 99295 10077 97653 90043 90416 97966 30565 20901 58660 94186 41307 20633 50489 46149 44397 39733 44867 87142 66405 7912 30261 58478 1846 60673 76532 52480 10858 4615 98765 19509 15366 63823 84969 7315 5443 65520 26356 29302 8052 45804 81536 5754 86365 9520 86955 58257 93660 15167 33508 15569 30359 40594 39513 29467 14232 61288 84960 15629 98918 65189 24617 15075 4228 80686 33989 88984 36653 57422 95545 42397 5464 46711 17376 24360 74816 76297 21322 15350 29544 18792 87006 17258 36197 38783 54320 81595 98395 13808 4141 19750 49432 4614 18437 64910 86668 39977 71555 8604 16602 33667 68285 35658 93624 2465 9690 487 83278 51379 52141 87961 97101 42350 81019 3699 92684 31285 23063 72298 5847 56287 51348 75788 63515 60669 14570 69602 22795 53656 4570 58334 43584 96322 23219 23907 62539 13467 36718 64260 74656 35187 34754 78530 20733 43299 34652 81982 90502 61477 28544 1359 98648 53249 47113 17738 29382 17171 32604 92635 60472 79383 83731 53188 64420 87910 45372 90752 43751 84118 84668 85562 29285 52235 28299 44998 3523 69552 77502 68522 44601 59701 63196 41258 61579 19090 10743 1810 34936 13450 20704 74363 17049 10432 82774 57577 18685 72529 46172 4799 54567 50699 2035 52620 17440 93979 69257 78355 16293 60595 44922 93855 60440 71469 21911 12417 24188 13214 52331 87550 79365 13461 2937 57700 95344 65324 26458 39316 64090 40987 37305 58428 64056 7253 75310 96460 82047 17800 86622 99698 57346 12493 5823 11321 6033 3281 90505 37976 27328 30759 89907 80187 86677 2879 61135 71154 89995 28865 49077 10046 22983 77700 86620 51691 23336 61381 31038 51230 49842 81904 91988 59665 70936 23024 13293 72048 77622 67487 49448 53693 87175 11568 26598 49204 76927 69660 94552 3216 71266 57019 59561 55656 88247 12285 61109 95577 18292 90688 57079 74141 21922 86092 23530 67999 28274 3732 10387 89042 22854 43774 30453 85972 41176 48083 18702 89409 40571 66578 9411 41617 70593 27149 86372 90185 74572 62778 95849 69006 63380 73909 41689 38544 5098 47524 95784 46060 62929 73218 54922 15052 45429 84695 99412 16569 39786 2304 53695 7220 60896 22417 85657 74874 16660 74327 81152 36823 43987 18863 23653 5094 61247 66900 20621 37332 51981 39418 63646 30581 93130 55275 82697 87606 39587 79349 36134 34823 32207 18295 91222 73122 95676 69404 84691 30348 3304 66397 92042 5382 56886 3052 69754 67163 78346 11035 32509 49639 97269 73706 77019 81224 3315 60170 33352 11195 79608 15708 48476 45881 33032 71143 45267 62105 90219 43482 54642 88322 76468 59318 20939 33733 19004 53625 54032 43252 50787 81704 62238 6822 70547 52469 27818 1923 43620 7707 53758 54728 71746 8182 27790 60457 22297 7317 97943 67882 21840 95581 31246 72802 36356 16834 78122 38748 83555 20541 46527 45951 16855 87932 47485 56029 54727 11780 38836 58616 64484 75780 79122 63396 46905 56973 17520 752 92346 56528 12342 65286 67627 71709 80839 97500 68292 2880 33413 72316 49604 17637 80726 58555 26942 85621 71822 32001 4850 96201 89806 21657 10295 8246 78250 53950 21446 96518 96991 35253 99958 14264 35097 15227 79477 40587 4098 48808 64547 67707 11396 26734 84645 48760 91106 30732 55803 91679 95426 28980 12970 46957 84580 72427 98259 59322 93461 61892 86570 95525 43023 94667 9742 48555 6951 93055 52360 57894 97556 19782 63985 73044 10742 65872 26547 89039 72528 86495 7477 43950 7836 37853 12767 92503 30709 35020 88194 87150 93657 88126 26910 20526 44754 9253 38762 85597 7769 24868 78470 36514 10216 83860 71163 31704 68980 66058 80099 68214 73128 32024 50932 47398 11321 19788 70772 37221 32130 35899 76559 43834 3133 65824 59591 16999 38977 71683 42745 23588 94566 54373 77966 95684 81753 70107 87880 21864 46258 45160 58129 42423 81364 46488 16359 25624 17154 60296 32920 37917 16270 23590 41782 66220 57798 33074 71722 99949 75038 20637 58765 6231 99878 260 40530 20663 45420 70014 72268 31714 69937 77467 60509 6014 51670 97714 32640 54318 46297 73524 16808 27078 27207 94426 19032 86605 69302 28575 23117 5138 21299 85612 53720 58133 80332 99808 36865 80662 79907 22069 12155 73472 65340 6405 51083 35195 24506 7058 69436 28037 35675 92301 61092 45959 67731 77707 55897 27616 70998 40190 38264 4433 38524 78758 46731 75601 45696 29288 86488 79057 64153 76314 8450 4744 60669 54924 46907 21563 70 16621 24326 99105 60058 91578 82152 50463 18288 31231 90716 64430 25880 42731 9497 88065 51682 90062 20762 995 26935 3433 27230 20520 48261 63552 15109 49873 62471 69414 68012 26161 42198 552 14596 55693 42251 26270 74654 42441 2167 43460 9338 8584 15109 4410 77671 95828 62896 83874 99445 54160 99011 54856 36139 97891 34214 56473 71390 31118 79873 67031 79872 81276 1976 55678 20264 24962 48817 65407 65992 51472 46527 94711 74493 70722 10525 72394 50299 47123 75978 41010 62830 67465 19414 28705 68479 9536 69624 5023 7514 82098 84632 37138 22555 97693 63262 55977 7526 33244 8093 14856 27213 59548 43371 12459 48100 33961 3277 73380 69950 20929 58957 72088 73404 78964 27114 10382 9202 72148 68426 39035 13056 31050 96641 40713 30395 84767 2631 94833 66311 9679 91480 36760 55523 29828 70146 37636 56461 82516 61348 55543 41296 16285 29515 63823 84705 835 58320 6610 8372 11336 29613 19846 16216 68898 4460 14567 2574 68186 48654 13732 97636 66961 78928 90357 4518 41126 18604 38912 61109 4211 40849 44391 13872 51706 24991 29280 26522 96649 32978 34566 43342 86946 93660 45312 49110 33507 84001 66566 84313 68223 5546 2253 72823 20163 94214 16984 16101 7419 46827 52366 13229 21253 69747 53243 79805 58295 95806 96849 11353 5666 4672 66380 57021 18884 71827 32156 1644 76982 72486 59997 80492 82274 93297 37395 23762 52362 89561 3041 89398 97089 17736 74699 63663 54539 51802 3496 83285 65539 18623 41120 32009 62784 77385 503 12693 29888 72062 3608 10027 8121 14377 47397 3527 55869 99594 80042 35991 72355 23653 80196 20691 16069 36864 28710 10348 8541 42528 67979 53997 98723 84834 2269 67449 39684 96514 35647 48400 33613 28659 51465 95546 35125 31974 36320 89855 87775 96026 72640 11209 39940 94386 98189 71893 78923 87604 57338 95618 19000 93957 81238 54993 1347 58950 7954 60599 27710 88458 26582 15973 61913 46176 68570 89194 91279 67230 3569 63208 24777 10621 74479 16245 15299 91525 5045 37682 40823 73395 28365 45322 88329 58155 25465 28326 18434 44080 7589 45560 65382 33871 15459 69807 86595 53224 15144 5276 21861 75140 5452 44604 51947 62064 17707 73279 91194 13222 74253 41992 59914 87990 88802 42701 19794 66737 46911 52391 10087 43157 34272 21733 62251 82310 3598 75435 96176 60522 76297 25689 43128 64947 50473 36515 42397 61024 40520 52359 42469 34576 19757 97467 30483 22279 74286 64163 54314 74057 41412 24640 80050 33764 91676 56967 4096 88487 2377 60582 43419 12729 18533 68130 92464 15754 58071 4490 59142 8572 9924 139 62474 18779 45610 5813 3765 63655 71629 40660 22579 89099 70486 88644 47928 89541 28357 16566 58825 28036 30944 17929 60829 69534 96376 65269 81763 93296 82754 87893 43793 94888 12484 97503 48971 91970 63280 93189 63704 60620 88851 91163 71783 11920 54273 51816 10537 78432 57521 90602 18120 32966 68561 30821 72262 31213 46151 76134 79668 84702 95343 41558 95573 71103 20586 79156 15407 81120 43224 17114 28023 65352 9697 14493 20885 99262 95876 9121 5510 41631 38729 90828 8728 11147 37599 16736 77189 85841 96742 87409 29556 93963 41897 11207 71701 84615 96721 70339 15454 84520 25059 84940 88493 97313 19773 85037 51715 81005 78238 74954 19839 86573 74933 68142 6012 53222 72895 69373 44726 93116 32724 94257 52405 31692 7639 43341 61429 8183 68663 37880 92975 17733 71323 70692 27707 98706 44548 33895 46491 81289 80072 24870 2769 86676 26163 61720 31516 76834 69755 24469 12661 16549 15846 81238 79385 47210 49029 10464 9571 43542 1445 41783 52355 3185 30086 2094 75748 67020 99007 91094 90222 7364 79401 35725 44425 92793 74667 72611 13321 55776 44480 72803 72288 77904 60877 65868 66460 55066 95403 75007 35110 94762 86536 61517 52722 25199 16905 48283 66815 18544 4964 86395 54006 80988 62616 76756 51897 55198 9846 69404 49318 83447 47288 97765 6328 88152 90358 44287 58503 96564 70417 38110 76005 34927 83225 88536 24483 73347 26511 34859 9130 28035 32581 26430 36217 39580 75628 78449 50573 54711 74511 37669 61875 85739 41221 83836 84163 6474 53627 8889 59033 82926 43551 69527 77833 9897 34250 54827 41964 69941 48206 54586 65400 27914 84064 12605 55995 99214 57262 39193 31647 30211 11319 44116 42882 25142 99037 35566 83214 74229 95981 73803 25602 77447 99889 8524 77908 77689 69744 96245 40404 63022 31572 8947 12195 38654 79601 16513 85978 74256 30 58141 43933 31248 81958 66253 48545 43762 96419 66598 39301 89219 80100 10073 99572 83509 58201 43378 86732 1985 42931 43792 90580 82253 31901 70913 58301 29793 65823 74239 41651 24910 42571 36923 95076 76838 68826 68202 44266 57652 46794 52006 77664 63055 78127 4454 63618 86988 68526 49916 81176 47606 59258 13149 65653 12292 52269 28346 20753 82588 47112 98742 27390 44459 9676 76432 25472 34190 45826 90508 14383 67063 39115 66571 38668 26992 6048 36409 12231 96444 79255 72527 96204 35983 60512 78488 86763 9434 98732 9086 49153 61444 29852 35820 32701 77788 63430 5683 42299 66198 96747 21250 4102 46994 28819 36884 29407 71278 38684 42698 98286 91301 71462 44862 12901 70452 63882 47156 37914 10606 22379 42736 66678 55027 40906 47921 9135 83029 35789 96181 7075 3930 99264 4217 82444 83090 92953 90551 46133 49700 4039 59894 1441 30821 36903 93633 90681 53367 67596 59491 61792 39377 79651 37514 87284 82535 31797 66951 62970 22781 92156 85963 20128 3147 44756 51356 31333 29081 44368 38176 1411 77414 31901 91496 56791 65851 63523 81022 10022 670 61679 74925 40368 68590 62647 39528 78710 47661 74403 51617 3790 9863 10416 6489 91849 73499 15636 38500 30647 6382 33145 88817 1381 73393 41938 52616 90213 88168 45708 97551 61727 85637 73909 39421 92432 7142 22520 67026 92095 61881 64492 19117 11783 15509 49297 69754 43412 11086 52946 3427 92832 3688 7944 63001 51208 11379 52094 84754 79672 82155 41848 42581 18067 95538 70549 97832 53974 44707 28460 78998 50723 72844 67995 1733 58269 73884 1334 35622 22706 23259 13789 90169 45477 10201 19661 56175 42206 64272 97214 61959 81581 9932 1428 80351 57060 44033 16360 88600 78313 55801 2325 11506 96975 12805 28922 56977 98579 23284 21507 67414 9105 65783 7813 65034 59442 74061 88632 38848 33701 59469 78364 21461 91544 92555 88574 71101 72980 16384 22834 46274 58931 7946 16626 73703 7048 92465 47342 65020 42780 64277 75079 37115 69540 50387 105 39300 25162 20381 71784 53816 61605 55268 57675 20394 38408 56189 88970 24914 38592 155 28558 84888 78076 24955 71402 30965 64508 66616 52670 72085 55427 31546 44294 71946 96926 43653 82446 41002 4897 48009 11054 40012 23732 47568 49065 31930 20870 29299 8651 67651 34979 71428 56291 78344 51899 94770 74121 39334 25121 50293 94279 11013 29864 29079 28701 69706 2014 88952 17119 38472 91202 51470 93311 98006 54248 54557 9937 45977 89959 96768 78033 44484 57450 9230 30841 36996 57558 5157 80350 63979 59056 70375 24165 91646 46263 56354 20146 17380 81694 70729 69559 88622 79694 84528 79601 6270 40276 84735 89024 24108 20267 87577 86339 91672 15454 26126 66776 50427 84700 10321 52166 38706 42970 54445 17691 72271 37883 7847 43980 34176 98610 863 2363 97873 13622 62394 87483 25182 80179 37696 68316 3379 35208 1336 5528 42370 20220 88405 4544 3285 66197 45269 48255 95797 52100 64907 52655 37189 70525 34672 41383 90800 45930 14729 19075 59837 54605 8289 63253 9541 22741 25843 55513 42759 71098 63204 1057 64059 39981 88198 97567 78995 90548 53829 98571 61135 35559 62401 78554 66854 3819 55051 22447 44852 37199 92562 44373 79931 87467 13353 64186 29542 50960 84782 34303 60421 24007 1661 26571 18561 25681 57797 27639 97779 23332 19869 21264 81296 76702 14180 59368 3568 56851 42661 75475 30145 66593 51754 91411 17961 14666 48605 29346 17681 88321 62588 35381 58835 16675 87963 67433 55466 31747 70169 18637 96124 39088 96741 90034 60467 77077 36766 43601 70514 84396 34428 60859 89592 11949 57835 41143 54201 70041 46325 73088 40068 70677 9754 81903 48987 93117 4138 90504 9338 12720 85570 66173 26480 63365 10745 71855 4798 80796 72361 72052 76485 56454 67458 9914 25109 14382 85840 29195 66662 31815 12742 28898 46695 67693 87036 30869 32243 21423 75894 66975 67394 38402 31247 32768 47882 29682 69964 2972 74936 55884 5215 69561 26872 14453 60139 26020 12630 37548 87728 89221 68769 29792 90930 987 34085 17962 34276 30213 12573 25410 89493 68913 69327 28387 89230 85217 74790 18305 93817 94661 47765 53146 29917 93455 87895 20306 51557 92329 56531 99703 63555 74335 88023 46015 95895 49230 55041 15080 9752 97309 85295 58811 10253 69603 95431 60329 15120 28860 20894 95517 92882 68009 56362 25397 63006 2764 18107 680 17986 25726 5295 72544 94837 28871 52136 31745 54544 43315 26826 1353 38004 86700 35784 53510 48802 37999 34148 39277 30336 57919 61554 15035 9447 64739 13299 19677 92041 64249 98111 24869 43853 7367 88064 36275 39640 99406 53739 8867 19436 91318 51340 4781 44573 89402 74912 35543 68780 46739 4845 83462 96541 27513 50630 95555 51154 73698 62325 88266 74945 19525 30863 62310 53130 82605 86346 98991 19182 7388 44204 42644 58173 65810 20413 35400 95972 6463 16967 79455 62221 55985 49910 18049 91651 46395 23107 82133 67525 21072 46352 40050 66373 44034 16399 9078 94183 61091 77339 93073 47892 32484 71529 15532 15057 63898 958 19457 28723 42257 46933 98943 54260 62299 80250 11690 73590 93883 91936 55678 17082 51057 66981 81917 30311 42724 99208 67506 46022 64223 43615 78386 13043 7879 57558 66388 66979 17818 2124 88135 84517 72874 59775 96624 82082 58921 82501 99949 58813 26051 54912 11032 57118 71643 63036 31450 95064 69308 26130 62043 3739 16948 38866 66924 17888 55090 20857 60823 7119 49830 95166 56081 95831 14916 9729 41951 31423 678 56621 53481 56346 81884 78209 83209 18906 44793 62211 71225 79541 4896 94388 68172 86441 48156 21795 28186 67266 99724 64529 28969 21276 95632 27579 9647 36967 51576 50274 84182 81573 69013 24138 85478 90838 39506 35186 44776
20926 8986 55366 5802 5559 78895 19861 74788 47949 65289 64609 96677 96046 20764 11760 16345 33616 54140 80760 72842 54682 29574 33271 88779 46668 60239 24453 96699 21259 92476 46462 63158 66368 34073 32551 96411 25616 8399 77448 28821 79078 45226 16087 43851 52651 75098 55716 19645 29726 68823 66866 40237 92728 16663 56394 51003
