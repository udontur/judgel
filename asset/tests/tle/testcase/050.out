12099288414146
